Rakefile

```diff
@@ -1,4 +1,5 @@
 require "bundler/gem_tasks"
+require "rails"
```

Comment:
let's avoid this require and use both options `--skip-webpack-install --skip-javascript`